context/contextTypes.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  AllConversationsType,
+  ConversationCardType,
+} from '@/types/messagingTypes';
+
+export interface IConversationState {
```

Comment:
What does the "I" stand for here? If it's "initial" then I think better to replace with the full word

Reply:
I can't express how much I love this comment 🖤 

This is a case where we decided to deviate from a convention and use _interface_ instead of _type_ so the name is using the `C#` convention of prefixing the name with `I` to indicate it's an interface.

However, you're right, that naming is not really the best when we stop to think about so maybe it should be `ConversationStateInterface` or even just `ConversationStateType` anyway because it devs don't need to be thinking about that difference when implementing.

P.S.: as a fun note, I learned very recently that when we prefix a name with the type (like we did with IConversationState) it's called "Hungarian Notation" - nifty

Reply:
Added this change to a new commit 👍

Reply:
Thanks for the explanation :) I wonder why it's Hungarian...